packages/runtime/src/events/EventProxy.ts

```diff
@@ -16,7 +16,14 @@ import {
     PeerSharedAttributeSucceededEvent,
     RepositoryAttributeSucceededEvent
 } from "./consumption";
-import { MessageDeliveredEvent, MessageReceivedEvent, MessageSentEvent, PeerRelationshipTemplateLoadedEvent, RelationshipChangedEvent } from "./transport";
+import {
+    MessageDeliveredEvent,
+    MessageReceivedEvent,
+    MessageSentEvent,
+    MessageWasReadAtChangedEvent,
```

Comment:
Shouldn't this be called `MessageWasRead**And**ChangedEvent` ?

Reply:
the property of the `Message` is called `wasReadAt`. This event notifies for when this property was changed.

Reply:
ah, ok still a little bit confusing but also have no better idea